tests/test_breadcrumbs.py

```diff
@@ -37,41 +36,25 @@ def test_breadcrumbs(sentry_init, capture_events):
                     message="breadcrumb2", **add_breadcrumbs_kwargs
                 )
 
-                # Spans that create breadcrumbs automatically
-                with sentry_sdk.start_span(name="span3", op=OP.DB_REDIS) as span3:
```

Comment:
these were outdated @antonpirker since we don't auto-create breadcrumbs from spans now